vllm/attention/backends/flashinfer.py

```diff
@@ -340,7 +349,7 @@ def begin_forward(self):
                 self.page_size,
                 # Disable flashinfer's pos encoding and use vllm's rope.
                 pos_encoding_mode="NONE",
-                data_type=self.data_type)
+            )
```

Comment:
Why is this not needed anymore?

Reply:
The wrappers are throwing dispatch errors with Float8 datatype with the current flashinfer versions(0.1.4 and 0.1.5). I have a couple of future MRs planned. I will enable it if future releases of Flashinfer fix the dispatch api. Seems like there would be api name changes with flashinfer too.

Reply:
I see, this is sufficient for now and it seems we will likely need to address this in flashinfer first